internal/cluster/validator.go

```diff
@@ -281,6 +287,11 @@ func doesClusterNetworksMatchNetworkType(cluster *common.Cluster) bool {
 	})) && cluster.NetworkType != nil && swag.StringValue(cluster.NetworkType) != models.ClusterNetworkTypeOVNKubernetes
 }
 
+func doesAvailabilityModeMatchNetworkType(cluster *common.Cluster) bool {
```

Comment:
I don't think we should block, we just want the default for SNO to be OVNKuberntes

Reply:
the decision with PMs was to block

Reply:
@eranco74 The ticket says "_Allow OVN only as NetworkType_". What am I missing?

Reply:
OK, I thought we just want it as default.
But if the PMs want to block it so be it.